backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/util/Functions.java

```diff
@@ -0,0 +1,26 @@
+package ca.gov.dtsstn.cdcp.api.web.util;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ResponseStatusException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonpatch.JsonPatch;
+import com.github.fge.jsonpatch.JsonPatchException;
+
+
+@Component
+public class Functions {
```

Comment:
I sent you a message in Slack about a different (in my opinion, more robust) way to implement this.

Reply:
removed after #1701 was merged